.docs/Get-VSTeamWorkItemType.md

```diff
@@ -20,21 +20,78 @@
 Get-VSTeamWorkItemType -ProjectName test -WorkItemType 'Code Review Response'
 ```
 
-This command gets a single work item type.
+This command gets a single WorkItem type from the named project.
+
+### Example 2
+
+```powershell
+Get-VSTeamWorkItemType -ProcessTemplate Basic
+```
+
+This command gets a list of the WorkItems available to projects which use the 'Basic' Template.
```

Comment:
you mean work item types not WorkItems, right?

Reply:
yes. I'll change it.